### PR TITLE
Add README for save-api library

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Add README for save-api library

### CODE REVIEW

File: save-api/src/main/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Save Cloud API Library
+
+Current module provides the library for usage of Save Cloud API.
```
Comment (petertrr) [inline save-api/src/main/README.md:3]:
```suggestion
Current module provides the library for usage of Save Cloud API. The library is written in Kotlin using Ktor client and is available as a dependency for JVM projects.
```

File: save-api/src/main/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Module contain few main interfaces: Set of http requests in `RequestUtils.kt`, which contain the required
+requests for tool evaluation in Save Cloud system, and client, `SaveCloudClient.kt`,
+which combined in itself completed interface for execution submission. Basically, the only client
+is necessary for usage of Save Cloud API. It requires just a tiny configuration for running the process of execution submission.
+
+The four main characteristics for client are necessary for execution:
+
+* **Configuration for web client** \
+  Basically it's just the Save Cloud backend address and port. (`WebClientProperties`)
```
Comment (petertrr) [inline save-api/src/main/README.md:13]: > backend

Let's call it API server address or something similar

File: save-api/src/main/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* **Authorization data** \
+  Credentials for authorization in Save Cloud system (`Authorization.kt`)
+
+* **Execution type** Git or standard
+
+
+  Detailed description about these configurations could be found in
+  [Backend-API.md](/../../../save-backend/Backend-API.md)
+
+The example of cli application, which uses this library could be found in `save-api-cli` module,
```
Comment (petertrr) [inline save-api/src/main/README.md:26]:
```suggestion
The example of cli application, which uses this library could be found in [`save-api-cli` module](../../../save-api-cli),
```

File: save-api/src/main/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Save Cloud API Library
```
Comment (petertrr) [inline save-api/src/main/README.md:1]: Please move this file into `save-api`; documentation isn't usually placed in `src`.

**Review by petertrr — APPROVED** [review]

File: save-api/src/main/README.md (hunk 4 above)
Comment (kgevorkyan) [reply save-api/src/main/README.md]: Thanks, misplaced, fixed

File: save-api/src/main/README.md (hunk 3 above)
Comment (kgevorkyan) [reply save-api/src/main/README.md]: Changed

File: save-api/src/main/README.md (hunk 1 above)
Comment (kgevorkyan) [reply save-api/src/main/README.md]: Added

**Pull request merged**